flow/project.py

```diff
@@ -86,48 +86,58 @@
 {template_vars}
 
 Filter functions can be used to format template variables in a specific way.
-For example: {{ project.get_id() | captialize }}.
+For example: {{ project.get_id() | capitalize }}.
```

Comment:
This seems unrelated?

Reply:
It is, but it's such a minor typo fix that I was willing to put it into this PR.

Reply:
According to our [governance](https://signac.io/governance/) 

> Committers may commit small changes (documentation updates, code comments, CI configuration changes) without a pull request

I think this falls under this.

The reason that I am so anal about this particular rule, is because we generally squash all commits prior to merging PRs and that means that these unrelated changes are now convoluted with the actually relevant changes. This is highly confusing to anybody who is reading the git log and attempts to understand the motivation for a particular change.

Reply:
I'm with @csadorf on this one, I can't find the article now but if we are strict about things like this, git blame and other git features work best when we keep the git log/commit history concise and "right"

Reply:
Fair enough, I can revert this change then.